src/config.ts

```diff
@@ -11,6 +11,8 @@ import ConnectionsFileSchema from './domain/ConnectionsFileSchema';
 import { prepareInjectionConfigVars } from './service/configInjection';
 import { DEFAULT_CONFIG, CONFIG_FILENAME, CONNECTIONS_FILENAME, REQUIRED_ENV_KEYS } from './constants';
 
+const CONFIG_FILE_CONVENTION = /^sync-db(-?)[\w]*\.yml$/;
```

Comment:
This is probably going to match these as well. We can so a simple check 

Check `^sync-db-[\w]+\.yml$` and `sync-db.yml` separately.

```
sync-db-.yml
sync-dbasdfafd.yml
```